redbot/cogs/audio/converters.py

```diff
@@ -27,30 +27,33 @@
     "get_playlist_converter",
 ]
 
-_SCOPE_HELP: Final[
-    str
-] = """
+T_ = _
+_ = lambda s: s
+
+_SCOPE_HELP: Final[str] = _(
+    """
 Scope must be a valid version of one of the following:
 ​ ​ ​ ​ Global
 ​ ​ ​ ​ Guild
 ​ ​ ​ ​ User
 """
-_USER_HELP: Final[
-    str
-] = """
+)
+_USER_HELP: Final[str] = _(
+    """
 Author must be a valid version of one of the following:
 ​ ​ ​ ​ User ID
 ​ ​ ​ ​ User Mention
 ​ ​ ​ ​ User Name#123
 """
-_GUILD_HELP: Final[
-    str
-] = """
+_GUILD_HELP: Final[str] = _(
+    """
 Guild must be a valid version of one of the following:
 ​ ​ ​ ​ Guild ID
 ​ ​ ​ ​ Exact guild name
 """
```

Comment:
omg, I missed a parenthesis
```suggestion
"""
)
```